### PR TITLE
Add simple tab example

### LINKED ISSUE #25: Add an example for tab
We need an example for tab, so we can better understand all methods' behaviours.

Check other examples for guidance, feel free to ask for help if you need.

### PR DESCRIPTION
**Pull request by Ditti4**
This pull request is a submission for the Hacktoberfest and will fix #25.

The example simply showcases how to create a tab, add controls to it, how to enable and disable margins for tab pages (and what it looks like compared to each other) and how to get the total number of pages in a tab.

If you have any comments on how to improve the example, please feel free to tell me!

### CODE REVIEW

File: examples/tab.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+"""
+ Shows a simple tab with two pages, one margined, one not margined.
+
+"""
+
+from pylibui.core import App
+from pylibui.controls import Window, Tab, HorizontalBox, Label
+
```
Comment (nsgonultas) [inline examples/tab.py]: Can you please add a new line here?

> Surround top-level function and class definitions with two blank lines. 

https://www.python.org/dev/peps/pep-0008/#blank-lines

Then I can merge this, I think :tada:

Comment (Ditti4) [inline examples/tab.py]: Woops, overlooked that. On it!

Comment (nsgonultas) [inline examples/tab.py]: Can you do that without a new commit? `amend` + `push -f`?

Comment (Ditti4) [inline examples/tab.py]: I'll squash the two commits (saw your second comment too late). :)

**Comment by Ditti4** [comment]
Okay, wait a second. Still working on it. Rebasing accidentally removed the merges.

**Comment by joaoventura** [comment]
If it works fine, it seems good to go! 👍

**Comment by nsgonultas** [comment]
Thanks for PR @Ditti4 :octocat:

**Comment by Ditti4** [comment]
No problem! Thank you for helping me get my :shirt:!

Also, I like the ID of the merge commit.

**Comment by nsgonultas** [comment]
BTW, will we add a travis.yml @joaoventura, if we won't, can you please turn of travis thing?

**Comment by joaoventura** [comment]
Deleted the travis service from settings, hopefully it won't bother us anymore. In the future I will check how to make good use of it, like, for instance, running automatically the tests..

**Pull request merged**